docs/source/color_processing/color-processing-introduction/color-processing-introduction.rst

```diff
@@ -4,12 +4,10 @@ FTC Color Processing - Introduction
 Overview
 --------
 
-As of FTC SDK 10.1, the *FIRST* Tech Challenge Software Development Kit now
-includes some Color Processing features from OpenCV, a popular and powerful
-open-source library for vision processing.
+Version 10.1, of the FIRST Tech Challenge Software Development Kit
+contains some Color Processing features from OpenCV, a popular and
```

Comment:
"some" is extraneous.  The sentence is tighter if it is removed.  If you really want to leave the OpenCV advertisement, "Version 10.1, of the FIRST Tech Challenge Software Development Kit contains Color Processing features from OpenCV..."
 reads a tad more professionally.